libmultilabel/networks/caml.py

```diff
@@ -9,6 +9,14 @@
 
 class CAML(BaseModel):
     def __init__(self, config, embed_vecs):
+        """CAML (Convolutional Attention for Multi-Label classification)
+
```

Comment:
can mention/cite the CAML paper. Like line 31 at https://github.com/iliaschalkidis/lmtc-eurlex57k/blob/master/neural_networks/layers/attention.py